<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessModeWidget</class>
 <widget class="QWidget" name="ProcessModeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QRadioButton" name="normalModeRadio">
     <property name="text">
      <string>Normal Process</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="mergeSourcesRadio">
     <property name="text">
      <string>Merge Sources</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="extractAudioRadio">
     <property name="text">
      <string>Extract Audio</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
